src/main/model/querys_db.ts

```diff
@@ -65,6 +66,16 @@ export class Queries {
     updatedAt text NOT NULL,
     CONSTRAINT lock_pk PRIMARY KEY (project,username,hostname));`;
 
+  WORKSPACE_SEARCH_ITEM_GROUP_TABLE = `CREATE TABLE IF NOT EXISTS groupKeyword (
```

Comment:
Keep only one naming convention. For projects tables we use underscore, like this file_inventories (plural)